code/controllers/subsystem/traumas.dm

```diff
@@ -20,35 +20,81 @@ SUBSYSTEM_DEF(traumas)
-												 /obj/effect/decal/remains/human))
+												 /obj/effect/decal/remains/human)),
+						
+						"robots"   = typecacheof(list(/obj/machinery/computer/upload, /obj/item/aiModule/, /obj/item/device/aicard)),
+						
+						"doctors"   = typecacheof(list(/obj/item/clothing/under/rank/medical, /obj/item/clothing/under/rank/chemist, 
+						/obj/item/clothing/under/rank/nursesuit, /obj/item/clothing/under/rank/chief_medical_officer, 
+						/obj/item/reagent_containers/syringe, /obj/item/reagent_containers/pill/, /obj/item/reagent_containers/hypospray/, 
+						/obj/item/storage/firstaid, /obj/item/storage/pill_bottle, /obj/item/device/healthanalyzer/, 
+						/obj/structure/sign/departments/medbay, /obj/machinery/door/airlock/medical, /obj/machinery/sleeper, 
+						/obj/machinery/dna_scannernew, /obj/machinery/atmospherics/components/unary/cryo_cell)),
+						
+						"authority"   = typecacheof(list(/obj/item/clothing/under/rank/captain,  /obj/item/clothing/under/rank/head_of_personnel, 
+						/obj/item/clothing/under/rank/head_of_security, /obj/item/clothing/under/rank/research_director, 
+						/obj/item/clothing/under/rank/chief_medical_officer, /obj/item/clothing/under/rank/chief_engineer, 
+						/obj/item/clothing/under/rank/centcom_officer, /obj/item/clothing/under/rank/centcom_commander, 
+						/obj/item/melee/classic_baton/telescopic, /obj/item/card/id/silver, /obj/item/card/id/gold, 
+						/obj/item/card/id/captains_spare, /obj/item/card/id/centcom, /obj/machinery/door/airlock/command)),
+						
+						"the supernatural"  = typecacheof(list(/obj/structure/destructible/cult, /obj/item/tome, 
+						/obj/item/melee/cultblade/, /obj/item/twohanded/required/cult_bastard, /obj/item/restraints/legcuffs/bola/cult, 
+						/obj/item/clothing/suit/cultrobes/, /obj/item/clothing/suit/space/hardsuit/cult, 
+						/obj/item/clothing/suit/hooded/cultrobes/, /obj/item/clothing/head/hooded/cult_hoodie, /obj/effect/rune, 
+						/turf/open/floor/plasteel/cult, /turf/closed/wall/mineral/cult, /obj/item/stack/sheet/runed_metal, 
+						/obj/machinery/door/airlock/cult/, /obj/singularity/narsie, /obj/item/device/soulstone,
+						/obj/structure/destructible/clockwork/, /obj/item/clockwork/, /obj/item/clothing/suit/armor/clockwork, 
+						/obj/item/clothing/glasses/judicial_visor, /obj/effect/clockwork/sigil, /obj/item/stack/tile/brass, 
+						/turf/open/floor/clockwork, /turf/closed/wall/clockwork, /obj/machinery/door/airlock/clockwork/,
+						/obj/item/clothing/suit/wizrobe, /obj/item/clothing/head/wizard, /obj/item/spellbook, /obj/item/staff, 
+						/obj/item/clothing/suit/space/hardsuit/shielded/wizard, /obj/item/clothing/suit/space/hardsuit/wizard,
+						/obj/item/gun/magic/staff/, /obj/item/gun/magic/wand/,
+						/obj/item/nullrod, /obj/item/clothing/under/rank/chaplain))
 					   )
 	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)))
```

Comment:
Cult/clickwork walls and floor?

Reply:
Clockwork floors and walls are already a phobia. I separated out bloodcult, clockcult, magic and religion for the sake of readability

Reply:
Now that i see it you put them in the object list, which won't work and might runtime; move turfs to this section please

Reply:
Got it